appvia/kore
# DNS: Configure root domains

## Summary

Introduces a specific UI to manage root domains for Kore.


**Which issue(s) this PR resolves**:
Resolves #1297 

## Changes

Please detail any major changes and provide the minimal documentation about how these work or do alter the current behaviour.
Anyone who reads this should understand the intention of this PR without reading the code.

### Additional changes

* API support for filtering on service kind when listing team services.

## Screenshots

![image](https://user-images.githubusercontent.com/7505593/93353719-26226e80-f834-11ea-993b-a52438b144b5.png)

![image](https://user-images.githubusercontent.com/7505593/93353744-30dd0380-f834-11ea-9651-7b4832f98060.png)

## Testing

The following testing has been performed:

* Ensure kore has a set of AWS (EKS) credentials set up
* Open UI, log in as admin
* Select Configure -> DNS
* Add a new domain for kore to manage (e.g. `testytest.kore.appvia.io`)
* Observe that the domain is pending - you should see 'Action required'
* Find the action that is required by clicking on it, follow the instructions to set up the domain.
* After actions are performed, within some time (can take 5-30 minutes depending on DNS timeouts) the state should go to Success.
